picard/formats/asf.py

```diff
@@ -66,87 +72,395 @@ def pack_image(mime, data, type=3, description=""):
-        'musicip_puid': 'MusicIP/PUID',
-        'releasestatus': 'MusicBrainz/Album Status',
-        'releasetype': 'MusicBrainz/Album Type',
-        'releasecountry': 'MusicBrainz/Album Release Country',
-        'acoustid_id': 'Acoustid/Id',
-        'acoustid_fingerprint': 'Acoustid/Fingerprint',
-        'compilation': 'WM/IsCompilation',
-        'engineer': 'WM/Engineer',
-        'asin': 'ASIN',
-        'djmixer': 'WM/DJMixer',
-        'mixer': 'WM/Mixer',
-        'artists': 'WM/ARTISTS',
-        'work': 'WM/Work',
-        'website': 'WM/AuthorURL',
+    __load_tags = {
+        # Official MS tags in sequence https://msdn.microsoft.com/en-us/library/ms867702.aspx#wm_metadata_usage_topic4
+        'Author': 'artist',
+        'Copyright': 'copyright',
+        'Description': 'comment', # Multiplex comment* tags into this field
+        # Rating - per MS recommendation use WM/SharedUserRating
+        'Title': 'title',
+        'WM/AlbumArtist': 'albumartist',
+        'WM/AlbumCoverURL': 'web_coverart',
+        'WM/AlbumTitle': 'album',
+        'WM/AudioFileURL': 'web_official_release',
+        # WM/AudioSourceURL - unsupported
+        'WM/AuthorURL': 'web_official_artist',
+        'WM/BeatsPerMinute': 'bpm',
+        'WM/Category': 'category',
+        'WM/Codec': '~codec',
+        'WM/Composer': 'composer',
+        'WM/Conductor': 'conductor',
+        # WM/ContentDistributor - unsupported
+        'WM/ContentGroupDescription': 'grouping',
+        # WM/Director - unsupported
+        # WM/DVDID - unsupported
+        'WM/EncodedBy': 'encodedby',
+        'WM/EncodingSettings': 'encodersettings', # ID3 TSSE
+        'WM/EncodingTime': 'encodingtime',
+        'WM/Genre': 'genre',
+        # WM/GenreID - per MS recommendation use WM/Genre
+        'WM/InitialKey': 'key',
+        'WM/ISRC': 'isrc',
+        'WM/Language': 'language',
+        'WM/Lyrics': 'lyrics', # Only "lyrics" or first "lyrics:" will be included
+        'WM/Lyrics_Synchronised': '~lyrics_sync', # Special encode/decode
+        # WM/MCDI - unsupported
+        # WM/MediaClassPrimaryID - support to be added from Release Type
+        # WM/MediaClassSecondaryID - support to be added from Release Type
+        # WM/MediaCredits - unsupported
+        # WM/MediaIsDelay - unsupported
+        # WM/MediaIsFinale - unsupported
+        # WM/MediaIsLive - write only support to be added from Secondary Release Type
+        # WM/MediaIsPremiere - unsupported
+        # WM/MediaIsRepeat - unsupported
+        # WM/MediaIsSAP - unsupported
+        # WM/MediaIsSubtitled - unsupported
+        'WM/MediaIsStereo': '~stereo',
+        # WM/MediaNetworkAffiliation - unsupported
+        # WM/MediaOriginalBroadcastDateTime - unsupported
+        # WM/MediaOriginalChannel- unsupported
+        # WM/MediaStationCallSign - unsupported
+        # WM/MediaStationName - unsupported
+        'WM/ModifiedBy': 'remixer',
```

Comment:
I don't think https://msdn.microsoft.com/en-us/library/ms867702.aspx#wm_metadata_usage_modifiedby is a description of "remixer" because it's about modification of the file, not the song.